pkg/apply/prune/prune.go

```diff
@@ -166,14 +166,17 @@ func (po *PruneOptions) GetPruneObjs(inv inventory.InventoryInfo,
 	for _, pruneID := range pruneIds {
 		pruneObj, err := po.GetObject(pruneID)
 		if err != nil {
-			// If prune object is not in cluster, no need to prune it--skip.
-			if apierrors.IsNotFound(err) {
-				klog.V(4).Infof("prune obj not in cluster--skip (%s/%s)",
+			if meta.IsNoMatchError(err) {
+				klog.V(4).Infof("skip pruning obj %s/%s: no match for the kind %q in Group %q",
```

Comment:
"no match" sounds ambiguous. 
How about `skip pruning obj %s/%s: the resource type is unrecognized by the cluster (kind: %q, group %q)`.

Reply:
Done